lib/takeaway.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'menu'
+require_relative 'order'
+
+class Takeaway
+    attr_reader :menu, :order, :food_list
+    
+    def initialize(menu= Menu.new)
+        @menu = menu
+        @total_price = 0
+        @food_list = []
+        
+    end
+
+    def view
```

Comment:
The name of your methods are not super transparent. Maybe `view_menu` would be better here? Or `get_menu`? (view usually implies you do some formatting for the user that you are not doing here).